content/blog/2018-01-03-comm-call-v15.md

```diff
@@ -0,0 +1,41 @@
+---
+slug: "comm-call-v15"
+title: "Community Call - Writing Packages to Support Research Communities - zoon & greta"
+date: 2018-01-03
+authors:
+  - name: Stefanie Butland
+categories:
+  - blog
+topicid:
+tags:
+- ropensci
+- community
+- events
+- fellowships
+---
+
+**Join our [Community Call on Tuesday, January 30th](http://communitycalls.ropensci.org/)** (January 31 for our Australian friends)
+
+Nick Golding, 2017 rOpenSci Fellow, will talk about two R packages he has developed recently. [zoon](https://github.com/zoonproject/zoon) aims to promote open and reproducible research in ecological modeling by helping researchers share their code in a modular way and produce reproducible research artefacts. Nick has recently been trying to bootstrap a community around this idea and says this is a much harder problem.
+
+[greta](https://github.com/goldingn/greta) lets you write out and fit statistical models (like Stan or BUGS) but right in R. It uses tensorflow to make models scale to massive data, and is designed to be used and extended by other modeling packages. `greta` relies on some nice R tricks and lots of thinking about designing APIs for both users and developers.
+
+### Agenda
+
+1. Welcome (Stefanie Butland, rOpenSci Community Manager, 5 min)
+2. guest: Nick golding (35 min)
+3. Q & A (moderated by Scott Chamberlain, 20 min)
+
+</br>
+### Speaker bio
+
+**Nick Golding** is an applied computational statistician working as a Research Fellow in the [Quantitative & Applied Ecology Group](https://qaeco.com/) at the University of Melbourne, Australia. He develops statistical models and software to predict the distributions of species and human diseases like those caused by the Zika and Ebola viruses. He is particularly interested in improving these models with information about traits, mechanistic relationships and population dynamics. [Nick received an rOpenSci Fellowship](https://ropensci.org/blog/2016/12/12/ropensci-fellowship-zoon/) to continue developing a framework for building and sharing reproducible species distribution models.
+
+Nick on [GitHub](https://github.com/goldingn), [Twitter](https://twitter.com/_nickgolding_), [Google Scholar](https://scholar.google.co.uk/citations?user=peoal7wAAAAJ&hl)
+
+### Resources
+
+- [Announcing our first fellowship awarded to Dr. Nick Golding](https://ropensci.org/blog/2016/12/12/ropensci-fellowship-zoon/); blog post Dec 12, 2016
+- Zoon publication: Nick Golding, Tom August, Tim Lucas, David Gavaghan, E. Emiel van Loon, Greg McInerny (2017). The zoon r package for reproducible and shareable species distribution modeling. Methods Ecol Evol https://doi.org/10.1111/2041-210X.12858
+- Soon after the Community Call, we'll post the [video](https://vimeo.com/ropensci/videos)
```

Comment:
can we do https://vimeo.com/channels/rocommunitycalls instead ?

Reply:
I'll fix that